roles/bb-master/tasks/main.yml

```diff
@@ -75,23 +83,39 @@
   with_items: "{{ slave_master_allocations[bb_config_branch] }}"
   # NOTE(sa2ajj): in case a password changes, a restart is also required, but
   # this is for future improvements.
+  tags: bb-master
+
+- name: Make sure we have latest creds
+  become_user: "{{ bb_user }}"
+  copy:
+    dest: "{{ bb_master_dir }}/{{ metabbotcfg_short }}/{{item.filename}}"
+    content: "{{ item.content | to_json}}"
+    mode: "0600"
+  with_items:
+      - filename: hyper.pass
+        content: "{{ hyper }}"
```

Comment:
Assuming this is referring to the value in the vault, it might be nice to name it `hyper_password` (similarly `github_oauth_password`) just to be clear.